py2-deprecation.md

```diff
@@ -0,0 +1,132 @@
+- Feature Name: Python 2 Deprecation Schedule
+- Start Date: 2019-03-25
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+
+
+# Summary
+[summary]: #summary
+
+In light of Python 2.7 reaching its end of life (EOL) on Jan 1st 2020, Python 2
+will be deprecated from SaltStack no earlier then the Sodium release, that is
+either the Sodium release or a later release.
+
+
+# Motivation
+[motivation]: #motivation
+
+Python 2.7 will reach it’s official end of life (EOL) on [January 1st,
```

Comment:
`its`

Reply:
And apparently a bunch of other smart quotes. Fixed now ✨